### ISSUE TITLE
Fail to build UI as cesium folder already exists

### ISSUE THREAD

**Issue by M5oul** [duniter/duniter#624]
``` bash
29 Sep 12:45:04 - info: compiled 91 files into 4 files, copied 36 in 16.2 sec
Downloading file https://github.com/duniter/cesium/releases/download/v0.3.5/cesium-v0.3.5-web.zip...
Extracting Cesium to path /home/moul/projects/duniter_2nd/web-ui/cesium...
mv: impossible de déplacer 'cesium' vers 'public/cesium': Directory not empty

npm ERR! Linux 4.7.4-200.fc24.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "i"
npm ERR! node v4.5.0
npm ERR! npm  v2.15.9
npm ERR! code ELIFECYCLE
npm ERR! duniter-ui@0.1.25 postinstall: `node "node_modules/bower/bin/bower" install --allow-root && node "node_modules/brunch/bin/brunch" build && node cesium.js && mv cesium public/ && cp misc/cesium.js public/cesium/config.js`
npm ERR! Exit status 1
```

**Comment by M5oul** [comment]
The `rm -rf` solution couldn't works with non-Unix OSes.
We have to write a JavaScript which handle this two cases.

**Comment by M5oul** [comment]
Done, I dropped this commits.

**Comment by c-geek** [comment]
Fixed in duniter/duniter-ui@dc4739359b97400d0f55c8c6500573e49c59dd30
